### PR TITLE
Setup Travis CI/CD for AWS EB

### PR DESCRIPTION
**Pull request by rhernand3z**
Tested on AWS Digital Dev -- nypl-header-qa environment.

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -12,32 +12,60 @@ Pass in the following environment variables:
...
+
+* Create a new branch off the `development` branch
+* Send a PR pointing to the `development` branch upon completion
+* Once the PR is approved, it should be merged into the `development` branch
+* If there are several PR's in process, a release should be scheduled by merging all completed PR's into the `development` branch
+* When a release is to be deployed, the `development` branch will be merged into `master`
+* All releases merged into `master`, `MUST` be tagged and pushed to Github with their corresponding `version`
+
+## AWS Elastic Beanstalk Application Settings
+By using the `aws-cli`, developers can deploy the Homepage App to the desired AWS application environments listed below:
```
Comment (ktp242) [inline ReadMe.md]: I think here should be Header App, not Homepage App?

**Comment by EdwinGuzman** [comment]
@holingpoon want to take up this existing PR together?

**Comment by holingpoon** [comment]
@EdwinGuzman @ktp242 @kfriedman I am also taking over this PR to finish Travis AWS EB integration.

File: ReadMe.md (hunk 1 above)
Comment (holingpoon) [reply ReadMe.md]: Oops...copy and paste snafu! Will look through ReadMe.

**Comment by holingpoon** [comment]
@ktp242 @EdwinGuzman Please review. Thanks!

**Pull request merged**